src/test/fuzz/rbf.cpp

```diff
@@ -130,10 +130,10 @@ FUZZ_TARGET(package_rbf, .init = initialize_package_rbf)
             break;
         }
         pool.addUnchecked(parent_entry);
-        if (fuzzed_data_provider.ConsumeBool() && !child->vin.empty()) {
-            child->vin[0].prevout = COutPoint{mempool_txs.back().GetHash(), 0};
+        if (fuzzed_data_provider.ConsumeBool()) {
+            child.vin[0].prevout = COutPoint{mempool_txs.back().GetHash(), 0};
```

Comment:
Good that you removed the unnecessary `vin.empty()` check.

Until the first time `ConsumeBool()` returns true here, we'll be using a `child` with a default-initialized `vin[0]`. This was true before the change as well. Does some later code hook up default-constructed `CTxIn` to coinbase transactions or something?

It seems unnecessary to have created a `parent` at all if the `ConsumeBool()` returns false. But I guess it's good to test adding unrelated transactions between each `child`.